src/main.rs

```diff
@@ -32,15 +32,15 @@ struct Args {
 #[derive(Debug, Subcommand)]
 enum Commands {
   Start {
-    #[arg(short, long, value_parser, default_value = "~/.config/simpledns/dns.config.yaml")]
+    #[arg(short, long, value_parser, default_value = "./dns.config.yaml")]
```

Comment:
Setting it to the current directory means if simpledns is installed to someone's system, itll pull the config from the current working directory for whereever they run the command. Having an absolute path would be preferred to avoid confusion